### PR TITLE
Aded Action/Flash messages with javascrip timeout

### PR DESCRIPTION
**Pull request by danielsteenberg**

### CODE REVIEW

**Comment by atainton** [comment]
Where is the description, please make sure you add a description

File: app/assets/stylesheets/devise-edit.scss [hunk 1]
```diff
@@ -23,4 +24,13 @@ h2 {
 input {
 	background: linear-gradient(black, grey);
```
Comment (atainton) [inline app/assets/stylesheets/devise-edit.scss]: indentation is off

File: app/assets/stylesheets/nav.scss [hunk 2]
```diff
@@ -1,34 +1,39 @@
 .dropdown {
-	width: 5px;
```
Comment (atainton) [inline app/assets/stylesheets/nav.scss:2]: loving the red

File: app/controllers/user/sessions_controller.rb [hunk 3]
```diff
@@ -1,8 +1,19 @@
 class User::SessionsController < Devise::SessionsController
 
   def login
-    flash[:notice] = "Signed in"
```
Comment (atainton) [inline app/controllers/user/sessions_controller.rb:4]: why you removing this?

File: app/controllers/user_controller.rb [hunk 4]
```diff
@@ -1,7 +1,13 @@
 class UserController < ApplicationController
 
   def login
-		flash[:notice] = "Signed in successfully"
```
Comment (atainton) [inline app/controllers/user_controller.rb:4]: indentation is off

File: app/controllers/user_controller.rb [hunk 5]
```diff
@@ -1,7 +1,13 @@
...
+		user.find_by(email: params[:session][:email].downcase)
```
Comment (atainton) [inline app/controllers/user_controller.rb]: can you not call current user?

File: app/controllers/user_controller.rb [hunk 6]
```diff
@@ -1,7 +1,13 @@
...
+      	if user && user.authenticate(params[:session][:password])
+      	  flash[:success] = 'Signed in Successfully'
+
```
Comment (atainton) [inline app/controllers/user_controller.rb]: extra line

File: app/controllers/user_controller.rb [hunk 7]
```diff
@@ -1,7 +1,13 @@
 class UserController < ApplicationController
 
   def login
-		flash[:notice] = "Signed in successfully"
+		user.find_by(email: params[:session][:email].downcase)
+      	if user && user.authenticate(params[:session][:password])
+      	  flash[:success] = 'Signed in Successfully'
```
Comment (atainton) [inline app/controllers/user_controller.rb]: is this not meant to render something or redirect?

File: app/controllers/welcome_controller.rb [hunk 8]
```diff
@@ -5,4 +5,8 @@ def index
 		
   end
 
+  def show
+
```
Comment (atainton) [inline app/controllers/welcome_controller.rb]: please removed line

File: app/assets/stylesheets/devise-edit.scss (hunk 1 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/devise-edit.scss]: Complete

File: app/assets/stylesheets/nav.scss (hunk 2 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/nav.scss:2]: hahaha thanks buddy, what you need me to do in this sheet?

File: app/controllers/user/sessions_controller.rb (hunk 3 above)
Comment (danielsteenberg) [inline app/controllers/user/sessions_controller.rb:4]: not using it in that controller at the moment, using it in the users controller for when user sighs in successfully or not. Should this be done in the Sessions controller and not the users controller?

File: app/controllers/user_controller.rb (hunk 4 above)
Comment (danielsteenberg) [inline app/controllers/user_controller.rb:4]: complete

File: app/controllers/user_controller.rb (hunk 7 above)
Comment (danielsteenberg) [inline app/controllers/user_controller.rb]: added rendering so will render back to the home page

File: app/controllers/user_controller.rb (hunk 6 above)
Comment (danielsteenberg) [inline app/controllers/user_controller.rb]: sorted

File: app/controllers/welcome_controller.rb (hunk 8 above)
Comment (danielsteenberg) [inline app/controllers/welcome_controller.rb]: removed

File: app/assets/stylesheets/nav.scss (hunk 2 above)
Comment (atainton) [inline app/assets/stylesheets/nav.scss:2]: nothing, just happy to see the indentation fix

File: app/assets/javascripts/application.js [hunk 9]
```diff
@@ -15,4 +15,5 @@
 //= require jquery_ujs
 //= require global
 //= require turbolinks
+//= require action-slider
```
Comment (atainton) [inline app/assets/javascripts/application.js:18]: why is this here?

Comment (atainton) [inline app/assets/javascripts/application.js:18]: are you using it?

Comment (danielsteenberg) [inline app/assets/javascripts/application.js:18]: Yes i'm using it for my action messages to disappear after 4 seconds of signing in

**Comment by atainton** [comment]
:

**Comment by atainton** [comment]
:+1:
